### PR TITLE
[desktop] Portuguese translation updated

### PR DESCRIPTION
**Pull request by alejandroarturom**

### CODE REVIEW

File: desktop/lib/messagebundles/pt.po [hunk 1]
```diff
@@ -1415,118 +1412,105 @@ msgstr "Completo"
...
-#, fuzzy
 msgid "Total size"
-msgstr "Upstream Total:"
+msgstr "Tamanho total"
 
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/frostwire/gui/components/transfers/TransferDetailGeneral.java:130
-#, fuzzy
 msgid "Upload speed limit"
-msgstr "Velocidade de upload:"
+msgstr "Carregar limite de velocidade"
```
Comment (gubatron) [inline desktop/lib/messagebundles/pt.po]: nope.
No deberia ser algo asi como:

"Limite de velocidad de Carrega"?

File: desktop/lib/messagebundles/pt.po [hunk 2]
```diff
@@ -1415,118 +1412,105 @@ msgstr "Completo"
...
-#, fuzzy
 msgid "Share ratio"
-msgstr "Compartilhar Taxa"
+msgstr "Proporção de ações"
 
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/frostwire/gui/components/transfers/TransferDetailGeneral.java:146
-#, fuzzy
 msgid "Save location"
-msgstr "Compartilhar Taxa"
+msgstr "Salvar localização"
```
Comment (gubatron) [inline desktop/lib/messagebundles/pt.po]: perhaps it'd be better: `Localização no disco`

File: desktop/lib/messagebundles/pt.po (hunk 1 above)
Comment (gubatron) [reply desktop/lib/messagebundles/pt.po]: `Limite de Velocidade de upload'

File: desktop/lib/messagebundles/pt.po [hunk 3]
```diff
@@ -1415,118 +1412,105 @@ msgstr "Completo"
...
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/frostwire/gui/components/transfers/TransferDetailGeneral.java:152
-#, fuzzy
 msgid "InfoHash"
-msgstr "Copiar Infohash"
+msgstr "Infohash"
 
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/frostwire/gui/components/transfers/TransferDetailGeneral.java:156
 msgid "Magnet URL"
-msgstr ""
+msgstr "URL magnético"
```
Comment (gubatron) [inline desktop/lib/messagebundles/pt.po]: I think this is better left as "Magnet URL" or "URL de Magnet"

Comment (alejandroarturom) [reply desktop/lib/messagebundles/pt.po]: Look https://www.tecmundo.com.br/torrent/4387-o-que-e-link-magnetico-.htm :)

Comment (gubatron) [reply desktop/lib/messagebundles/pt.po]: I just think it's like translating "Torrent" or "Download" which we don't.
I'm also not sure of the tech pro-efficiency of that writer.

Comment (gubatron) [reply desktop/lib/messagebundles/pt.po]: It should also be capitalized as "Magnet".

And another point to my side, it's not a "Magnetic Link" in English either, which would be correct. It's a "Magnet"

Comment (gubatron) [reply desktop/lib/messagebundles/pt.po]: if anything in spanish/portuguese the translation would be "URL de Magneto"

File: desktop/lib/messagebundles/pt_BR.po [hunk 4]
```diff
@@ -1402,118 +1399,105 @@ msgstr "Completo"
...
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/frostwire/gui/components/transfers/TransferDetailGeneral.java:152
-#, fuzzy
 msgid "InfoHash"
-msgstr "Copiar Infohash"
+msgstr "Infohash"
 
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/frostwire/gui/components/transfers/TransferDetailGeneral.java:156
 msgid "Magnet URL"
-msgstr ""
+msgstr "URL magnético"
```
Comment (gubatron) [inline desktop/lib/messagebundles/pt_BR.po]: same here

**Pull request merged**